bin/main.rb

```diff
@@ -1,3 +1,78 @@
 #!/usr/bin/env ruby
+# Welcome user to the game
+puts 'Welcome to Tic Tac Toe!'
+# create game board
+class Board
+  def initialize
+    @board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+  end
 
-puts 'Hello World!'
+  # display board
+  def display_board
+    puts
+    "\n#{@board[0]} | #{@board[1]} | #{@board[2]}"\
+    "\n---------"\
+    "\n#{@board[3]} | #{@board[4]} | #{@board[5]}"\
+    "\n---------"\
+    "\n#{@board[6]} | #{@board[7]} | #{@board[8]} "
+  end
+
+  # assigning player chips
+  def board_update(choice, player)
+    choice -= 1
+    @board[choice] = (player == 'one' ? 'x' : 'o')
+  end
+end
+
+class Instructions < Board
+  # validate player choice
+  def player_choice?(choice)
+    choice.between?(1, 9)
+  end
+
+  # make a move and validate it
+  def pos_taken?(choice)
+    @board[choice - 1] == 'x' || @board[choice - 1] == 'o'
+  end
+
+  # game informs player if selected move is a winning move
+  def win?
+    "\nPlayer #{player} is our Tic Tac Toe Master!"
+  end
+
+  # game informs player if selected move is a draw
+  def draw
+    "\n Its a draw!"
+  end
+end
+
+game = Instructions.new
+game.display_board
+
+count = 0
+# game repeats all actions for player's moves
+while count <= 1
+  %w[one two].each do |i|
+    player = i
+
+    loop do
+      puts("\nPlayer #{i} It's your turn\n(choose between 1-9):")
+      choice = STDIN.gets.chomp.to_i
```

Comment:
STDIN it's not needed. You could directly use `gets.chomp`

Reply:
<img width="1440" alt="Screenshot 2020-04-24 at 11 45 23" src="https://user-images.githubusercontent.com/25789009/80193261-2752c500-8621-11ea-841e-2d40da93cd0b.png">
 Kindly confirm as seen below  in both my partner and I's screen the board displays after every move